bin/pull

```diff
@@ -40,12 +40,13 @@ fi
 
 ############## Error handling
 
+TEMP_DIR=$(mktemp -d)
 function report {
   if [ -f "$LOG" ]; then
-    content=$(tail -n 20 "$LOG")
+    content=$(tail -n 50 "$LOG")
   fi
 
-  message="performance test failed:
+  message="Performance test failed:
```

Comment:
I'd say, let's make that 1000 lines and put it within

```
<details>
  <summary>Logs</summary>

$CONTENT

</details>
```


<details>
  <summary>Logs</summary>

$CONTENT

</details>